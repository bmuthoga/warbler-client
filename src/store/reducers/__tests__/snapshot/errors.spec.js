import errorReducer from '../../errors'
import { ADD_ERROR, REMOVE_ERROR } from '../../../actionTypes'

describe('Error Reducer Snapshot Tests', () => {
  it('should return the initial state', () => {
    expect(errorReducer(undefined, {})).toMatchSnapshot()
  })

  it('should handle ADD_ERROR', () => {
    const error = 'Sample error message'
    expect(errorReducer(undefined, { type: ADD_ERROR, error })).toMatchSnapshot()
  })

  it('should handle REMOVE_ERROR', () => {
    expect(errorReducer(undefined, { type: REMOVE_ERROR })).toMatchSnapshot()
  })
})
