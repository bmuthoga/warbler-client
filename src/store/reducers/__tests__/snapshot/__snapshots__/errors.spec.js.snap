// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Reducer Snapshot Tests should handle ADD_ERROR 1`] = `
Object {
  "message": "Sample error message",
}
`;

exports[`Error Reducer Snapshot Tests should handle REMOVE_ERROR 1`] = `
Object {
  "message": null,
}
`;

exports[`Error Reducer Snapshot Tests should return the initial state 1`] = `
Object {
  "message": null,
}
`;
